### DISCUSSION TITLE
I having trouble understanding how to install an external RA-only service

### DISCUSSION THREAD

**Discussion by bugz8unny69** [Keyfactor/ejbca-ce#278] [Q&A] [answered]
Hi community,

I'm running EJBCA in my home lab. And I just recently discovered going through the documentation that is secure to run RA externally. The question here is, how? The documentation [here](https://doc.primekey.com/ejbca/ejbca-installation/installing-ejbca/installing-ejbca-as-an-ra-or-va) isn't very clear. Should I be following [this](https://doc.primekey.com/ejbca/ejbca-installation/installing-ejbca/install-ejbca-as-a-ca-without-a-management-ca) first? If so, how do I copy over the Keystore from a docker container? I appreciate any assistance on this matter.

Regards,
Horace

**EDIT:** I found https://certs.advania.is/doc/kara.html and will give this a shot.

**Comment by primetomas** [comment, marked as answer]
Don't go with the advania link, it is for an EJBCA version that is soon 6 years old.

You are correct about the two links you provided. One thing that was not too clear is that to run the EJBCA RA externally you need Peer Connectore, which is an Enterprise feature. The notice was hidden in the [Peers documentation](https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/peer-systems). I'll ask to make that documentation more clear.

Regards,
Tomas

**Reply by primetomas** [reply]
The advania link is actually the "same" as the newer documentation, also using "Peer connection", just that that documentation is very old.

**Reply by bugz8unny69** [reply]
Thanks for informing me that it's an enterprise feature. That forgo any future headaches.
